src/encoder.rs

```diff
@@ -6,16 +6,16 @@ use std::{borrow, error, fmt, io, mem, ops, result};
 use crc32fast::Hasher as Crc32;
 use flate2::write::ZlibEncoder;
 
-use crate::chunk::{self, ChunkType};
-use crate::common::{
-    AnimationControl, BitDepth, BlendOp, BytesPerPixel, ColorType, Compression, DisposeOp,
-    FrameControl, Info, ParameterError, ParameterErrorKind, ScaledFloat,
+use crate::{
+    chunk::{self, ChunkType},
+    common::{
+        AnimationControl, BitDepth, BlendOp, BytesPerPixel, ColorType, Compression, DisposeOp,
+        FrameControl, Info, ParameterError, ParameterErrorKind, ScaledFloat,
+    },
+    filter::{filter, AdaptiveFilterType, FilterType},
+    text_metadata::{EncodableTextChunk, ITXtChunk, TEXtChunk, TextEncodingError, ZTXtChunk},
+    traits::WriteBytesExt,
 };
-use crate::filter::{filter, AdaptiveFilterType, FilterType};
-use crate::text_metadata::{
-    EncodableTextChunk, ITXtChunk, TEXtChunk, TextEncodingError, ZTXtChunk,
-};
-use crate::traits::WriteBytesExt;
 
```

Comment:
It was automatically formatted that way.

Reply:
Interesting, the `rustfmt` didn't complain so this must be consistent with something. Maybe we didn't have a reformat since bumping the msrv and it applies a different style to different editions? I don't know the first thing about rustfmt internals or its development :D

Reply:
Would be nice to double check whether the reformat is demanded by rustfmt or merely tolerated by it.

Reply:
Trying rustfmt locally, it seems that it will accept either version without changing them. Given that, I'd prefer to avoid the churn of changing all the imports in this PR